src/applications/hca/config/form.js

```diff
@@ -407,6 +406,15 @@ const formConfig = {
                 },
               },
             }),
+            'view:doesPermanentAddressMatchMailing': {
+              'ui:title':
+                'Is your home address the same as your mailing address?',
+              'ui:widget': 'yesNo',
+              'ui:required': formData => formData['view:hasMultipleAddress'],
+              'ui:options': {
+                hideIf: formData => !formData['view:hasMultipleAddress'],
```

Comment:
Yes/no question that show/hides second address. This UI is also conditionally rendered based on flipper data on line 415.